libr/io/p/io_gprobe.c

```diff
@@ -47,12 +58,125 @@ enum {
 	GPROBE_ACK = 0x0c,
 	GPROBE_RESET = 0x20,
 	GPROBE_GET_DEVICE_ID = 0x30,
+	GPROBE_GET_INFORMATION = 0x40,
 	GPROBE_RAM_READ_2 = 0x52,
 	GPROBE_RAM_WRITE_2 = 0x53,
 	GPROBE_RUN_CODE_2 = 0x54,
 };
 
-static int sp_close (struct sp_port *port) {
+#if __UNIX__
```

Comment:
I don't know the context, but why this condition? Why should this plugin not be supported on Windows?

Reply:
because of ioctl, /dev/ and nanosleep()